src/yarpmanager/src-manager/clusterWidget.cpp

```diff
@@ -477,6 +485,20 @@ bool ClusterWidget::checkNode(const string &name)
         portname = "/" + portname;
     }
 
+    if (!NetworkBase::checkNetwork(2.0))
+    {
+        yError()<<"Manager::yarpserver is not running...";
```

Comment:
Personal comment: I am not a big fan of the `...` at the error of the messages, they do not seem very professional to me. 
Second comment: why `Manager` at the beginning of the message? If this is the message of the `yarpmanager` executable, I would put the `yarpmanager` string for clarity. Furthermore, why you use the `::` as a separator, that seems to hint to C++ namespaces? I would just write `yarpmanager: yarpserver is not running.` .